DEPRECATIONS.md

```diff
@@ -11,6 +11,7 @@ The following is a list of deprecations, according to the [Deprecation Policy](h
 | DEPPS5 | Config option `allowClientClassCreation` defaults to `false` | [#7925](https://github.com/parse-community/parse-server/pull/7925)   | 5.3.0 (2022)                    | 7.0.0 (2024)                    | deprecated            | -     |
 | DEPPS6 | Auth providers disabled by default | [#7953](https://github.com/parse-community/parse-server/pull/7953)   | 5.3.0 (2022)                    | 7.0.0 (2024)                    | deprecated            | -     |
 | DEPPS7 | Remove file trigger syntax `Parse.Cloud.beforeSaveFile((request) => {})` | [#7966](https://github.com/parse-community/parse-server/pull/7966)   | 5.3.0 (2022)                    | 7.0.0 (2024)                    | deprecated            | -     |
+| DEPPS8 | Allow login with expired authData token | [#7079](https://github.com/parse-community/parse-server/pull/7079)   | 5.3.0 (2022)                    | 7.0.0 (2024)                    | deprecated            | -     |
```

Comment:
The deprecation descriptions are all a bit unclear. It's unclear whether they describe how it's before or after the deprecation removal. For now let's continue to describe how it will be after the removal, e.g. "Login with expired 3rd party authentication token defaults to `false`."

Reply:
fixed